src/setOfPeople.js

```diff
@@ -39,6 +39,15 @@
  */
 function setOfPeople(people) {
   // write code here
+  const inRoom = new Set();
+  people.forEach(
+    (person) => {
```

Comment:
The line is too short to transfer to a new line

Reply:
Done